### PR TITLE
Added support for svmon client

### PR DESCRIPTION
**Pull request by Agupane**
Added support for svmon client, fork from the original pakiti proyect.

### CODE REVIEW

File: bin/pakiti-client [hunk 1]
```diff
@@ -238,8 +237,38 @@ sub find_packages ($) {
     my($data) = @_;
     my($cmd, $format, @list);
     my(@output);
-
-    # Red Hat packages
+
```
Comment (kouril) [inline bin/pakiti-client]: Please refrain from doing unnecessary changes, i.e. changing white spaces, etc.

File: bin/pakiti-client [hunk 2]
```diff
@@ -238,8 +237,38 @@ sub find_packages ($) {
...
-
-    # Red Hat packages
+	
+	#We check if we got called with svmon parameter
+	$cmd = $Option{"svmonpath"};	
+	#If we dont have the svmon execution parameter we use the default one
+	if(!$cmd) {
+		$cmd = 'svmon'
+	}
+	#If the path is ok we continue
```
Comment (kouril) [inline bin/pakiti-client]: Please don't comment obvious things (like all the three comments above)

File: bin/pakiti-client [hunk 3]
```diff
@@ -238,8 +237,38 @@ sub find_packages ($) {
...
+				#Format of line: 
+				#Site[/t]Endpoint[\t]OS[\t]Component[\t]CfgParameter[\n]
+				$line = "[".$line."]\n";
+				push(@list, $line);
+			}
+			#We add brackets at the beginning and on the end of the svmon data
+			unshift(@list,'{');
+			push(@list,'}');
+			$data->{packages} = join("",@list);
+			#$data->{packages} = join("", sort(@list));
```
Comment (kouril) [inline bin/pakiti-client]: please don't leave unused code there

File: bin/pakiti-client [hunk 4]
```diff
@@ -487,7 +521,6 @@ sub init () {
 
 sub main () {
     my(%data, $report);
-
```
Comment (kouril) [inline bin/pakiti-client:490]: There's a bunch of unnecessary new-line/blank-space changes above, please get rid of them.

File: bin/pakiti-client [hunk 5]
```diff
@@ -688,6 +725,10 @@ send the prepared report to the collecting server at the given URL
 
 set the path of the C<wget> command to use
 
+=item B<--svmon>, B<-sv>
```
Comment (kouril) [inline bin/pakiti-client]: - The short-option (prefixed with a single -) must be a single letter
- Please retain the alphabetic order of the CLI parameters

File: bin/pakiti-client [hunk 6]
```diff
@@ -238,8 +237,38 @@ sub find_packages ($) {
     my($data) = @_;
     my($cmd, $format, @list);
     my(@output);
-
-    # Red Hat packages
+	
+	#We check if we got called with svmon parameter
```
Comment (kouril) [inline bin/pakiti-client]: I believe you're using tab's to indent the code, please stick to using spaces (as in the rest of the code)

File: bin/pakiti-client [hunk 7]
```diff
@@ -238,8 +237,38 @@ sub find_packages ($) {
...
+	$cmd = $Option{"svmonpath"};	
+	#If we dont have the svmon execution parameter we use the default one
+	if(!$cmd) {
+		$cmd = 'svmon'
+	}
```
Comment (kouril) [inline bin/pakiti-client]: Please remove the default setting of svmon since that would break systems without the command. What you want to do is to rename the option to just svmon (i.e. to be the same as the command).

Comment (Agupane) [reply bin/pakiti-client]: I did not understood what you mean, if the systems doesn't have svmon, why they would use svmonpath as parameter?, Also it's not pretended that the users should always define by themselves a path of the installation of svmon or the command to call svmon (thats why there is a default svmon call). But they stil have the option in case that they need it. If the systems do not have svmon installed they won't use the svmon command, and in that case there is no a mandatory call of svmon from pakiti

@kouril
